src/test/java/com/example/bookstoreproject/persistence/book/BookStoreTest.java

```diff
@@ -0,0 +1,101 @@
+package com.example.bookstoreproject.persistence.book;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Optional;
+
+import static com.example.bookstoreproject.fakes.BookFakes.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class BookStoreTest {
+
+    @Mock
+    private BookRepository bookRepository;
+
+    @InjectMocks
+    private BookStore bookStore;
+
+    @Test
+    void shouldFindAll_OK() {
+        final var expected = buildBookEntities();
+        when(bookRepository.findAll()).thenReturn(expected);
+        assertEquals(expected.size(), bookStore.findAll().size());
+        verify(bookRepository).findAll();
+    }
+
+    @Test
+    void shouldFindById_OK() {
+        final var book = buildBookEntity();
```

Comment:
A unit test can have 3 steps

1. Given (Prepare something)
2. Then (Do an action, the thing we want to test)
3. Verify (assertion)

Add the empty lines between each step to maintain the readability

Reply:
DONE